src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -506,6 +508,95 @@ protected function initializeBundles()
     }
 
     /**
+     * Returns an array of bundles to register, with dependencies, ordered
+     *
+     * @uses appendDependenciesRecursively
+     *
+     * @return \Symfony\Component\HttpKernel\Bundle\BundleInterface[] An array of bundle instances.
```

Comment:
`@return BundleInterface[]` with the `use` statement at the top if not already set.